### DISCUSSION TITLE
Q: 文章列表页不加载评论数，但内页已经加载

### DISCUSSION THREAD

**Discussion by Angiin** [walinejs/waline#1945] [Q&A] [answered]
### Describe the bug | 问题描述

![)(5Z8P3}`LH(K O3JHVD$TJ](https://user-images.githubusercontent.com/17389304/233866459-a37567ca-104c-4176-b38c-cf7dda781219.png)
![6~H1JS DEEWA GX{GJ_E43M](https://user-images.githubusercontent.com/17389304/233866468-4b8db379-10fb-46d8-a5b8-a4c803973bfa.png)

不知道如何解决?

### Website URL | 问题网站

https://winisle.com

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

**Comment by Angiin** [comment, marked as answer]
已经独立解决。忘记引入了 Waline 客户端了。

官方的文档写的不是特别完整，自己做了一个完善：

```
{{/* Import the Waline client module */}}
<script type="module">
  import { commentCount } from 'https://unpkg.com/@waline/client@v2/dist/comment.mjs';

  {{/* Call the commentCount function with the serverURL and path values */}}
  commentCount({
    serverURL: 'https://example.com',
    path: '{{ .Permalink }}'
  });
</script>
```

**Reply by orangelegy** [reply]
你好，请教一下文章列表页统计评论数的 data-path 属性怎么填写，为什么我全是0。配置环境跟你一样的

---
Hello, please tell me how to fill in the data-path attribute of the statistical comments on the article list page, why I am all 0. The configuration environment is the same as yours

**Reply by Angiin** [reply]
这段代码直接改 serverURL 的值就可以了。不知道你的网站时基于什么的

```html
<script type="module">
  import { commentCount } from 'https://unpkg.com/@waline/client@v2/dist/comment.mjs';

  {{/* Call the commentCount function with the serverURL and path values */}}
  commentCount({
    serverURL: 'https://example.com',
    path: '{{ .Permalink }}'
  });
</script> 
```
